src/components/Table.js

```diff
@@ -81,12 +81,30 @@ const Table = ({ getSelectedPacket, packets, config }) => {
           setWindowStart(frameno);
           setWindowEnd(frameno + config.packetWindowSize);
         }
+        updateWindow(frameno);
         // FIXME: Use IndexDB
         localStorage.setItem(frameno, packet);
       }
     }
   }, [packets]);
 
+  /**
+   * Updates window if new packet frameno is greater than windowEnd (exceeds window) and
+   * new packets after windowEnd are greater than or equal to config.jumpSize (sizable packets after window)
+   * @param {number} frameno - frame number of received packet
+   */
+  const updateWindow = (frameno) => {
```

Comment:
Call it `mayBeUpdateWindow` already and return `true/false` or may be `didWindowUpdate`?